roles/init/tasks/postgres.yml

```diff
@@ -228,6 +228,28 @@
       {%- else -%}xlog
       {%- endif -%}
 
+
+# Find and set right HARP manager user. The var from config.yml takes precendence,
+# otherwise TPA will try to find the username from a configured cluster. If that's
+# not possible because of reasons, a default will be given.
+
+- name: Read current configuration file if exists
+  shell: "cat {{ harp_directory }}/config.yml"
+  register: config_yml
+  ignore_errors: true
+
+- set_fact:
+    current_harp_manager_user: "{{ item.split('user=')[1] | trim }}"
+
+  vars:
+    harp_config_struct: "{{ config_yml.stdout | from_yaml }}"
+    endpoints: "{{ harp_config_struct.dcs.endpoints }}"
+  loop: "{{ endpoints }}"
+  when: config_yml.rc == 0
+
+- set_fact:
```

Comment:
In this case, IIUC from the code, `harp_manager_user` will assume:

1- The value the user specified in `harp_manager_user` from `config.yml`; OR

2- The current user from the HARP Manager DSN, which in previous deployed clusters would be `postgres`; OR

3- The default specified by `default_harp_manager_user`, which is `superhman`.

I agree with the behavior.